python/tk_multi_publish2/dialog.py

```diff
@@ -113,8 +113,9 @@ def __init__(self, parent=None):
         self.ui.close.clicked.connect(self.close)
         self.ui.close.hide()
 
-        # overlays
+        # overlay
         self._overlay = SummaryOverlay(self.ui.main_frame)
+        self._overlay.publish_again_clicked.connect(self._publish_again_clicked)
```

Comment:
Did this get removed before and you're adding it back?

Reply:
Yes, that was me in a previous commit. 👎